### PR TITLE
Update crates version

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: atk/Cargo.toml [hunk 1]
```diff
@@ -31,7 +31,7 @@ features = ["dox"]
 libc = "0.2"
 bitflags = "1.0"
 ffi = { package = "atk-sys", path = "sys" }
-glib = { git = "https://github.com/gtk-rs/gtk-rs-core" }
+glib = "0.17"
```
Comment (sdroege) [inline atk/Cargo.toml:34]: You want to keep git

```suggestion
glib = { git =  "https://github.com/gtk-rs/gtk-rs-core", branch = "0.17", version = "0.17" }
```

etc

Comment (GuillaumeGomez) [reply atk/Cargo.toml:34]: Oh? Ok.

Comment (sdroege) [reply atk/Cargo.toml:34]: @bilelmoussaoui also missed that in gtk4

Comment (GuillaumeGomez) [reply atk/Cargo.toml]: Fixed. :)

**Review by sdroege — APPROVED** [review]

**Pull request merged**